src/app/views/tool_bar.tsx

```diff
@@ -468,33 +465,40 @@ export class MultiObjectButton extends ContextedComponent<
         {
           anchor: ReactDOM.findDOMNode(this.refButton) as Element,
           alignX: "end-outer",
-          alignY: "start-inner"
+          alignY: "start-inner",
         }
       );
     };
 
+    const onClick = () => {
+      if (!this.props.compact) {
+        return;
+      }
+      openContextMenu();
+    };
+
     return (
       <div
         className={classNames("charticulator__button-multi-tool", [
           "is-active",
-          this.isActive()
+          this.isActive(),
         ])}
       >
         <ObjectButton
-          ref={e => (this.refButton = e)}
+          ref={(e) => (this.refButton = e)}
           {...this.getSelectedTool()}
           onClick={onClick}
           compact={this.props.compact}
         />
         <span
           className="el-dropdown"
-          ref={e => {
+          ref={(e) => {
             if (this.props.compact) {
               return;
             }
             this.refButton = e as any;
           }}
-          onClick={() => {}}
+          onClick={!this.props.compact ? () => openContextMenu() : null}
```

Comment:
This could reference the `onClick` function you created earlier

Reply:
No. logic is opposite

Reply:
I updated functions to clarify logic